src/App.tsx

```diff
@@ -29,12 +29,28 @@ export const App = async (props: AppProps, container?: HTMLElement) => {
   konsole.log("BotChat.App props", props);
 
   // FEEDYOU generate user ID if not present in props, make sure its always string
+  
   props.user = {
     name: "Uživatel",
     ...props.user,
     id: props.user && props.user.id ? "" + props.user.id : MakeId(),
   };
 
+  if(body.config?.persistent === "user" || body.config?.persistent === "conversation" ){
+    if(sessionStorage.getItem("feedbotUserId")){
+      props.user.id = sessionStorage.getItem("feedbotUserId")
+    } else {
+      props.user.id = MakeId()
+      sessionStorage.setItem("feedbotUserId", props.user.id)
+    }
+    
```

Comment:
Here set new App & Chat prop `openUrlTarget: [new, same, same-domain]` + pls add to README.md